06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<p>I understand that the goal is to explore which features are prediction mutation status, and for that you are using the univariate correlation as you describe it at the end. I don't remember the prediction model you use, but I guess there could be also interactions between the predictor variables, right? That is, maybe univariate correlation is not capturing the genes/features that, in combination, help to predict the outcome.</p><p> </p><p>I know you are explicitly saying that you are using univariate correlations, but I wonder at what extent these interactions are useful to answer your question (if what I said makes sense).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Absolutely, I think it's a good starting point.